controllers/scan.js

```diff
@@ -2,46 +2,57 @@
 
 const sha1 = require("../sha1-utils");
 const HIBP = require("../hibp");
+const TIPS = require("../tips");
 
 
 async function post (req, res) {
   const emailHash = req.body.emailHash;
+  const whatToDo = "4";
```

Comment:
What does this do? I see we seem to be passing it to "scan.hbs" (L46 and L55 below), but I'm not seeing how/where it's _consumed_, and why it's a numbery string (or stringy number, if you prefer).

Reply:
hahaha it's a relic from when you first mentioned passing variables to svgs. removing!